frontend/src/domain/store/components/common/VideoCard/index.js

```diff
@@ -91,6 +91,7 @@ const ChannelAndDate = styled.span`
   color: var(--dark-gray);
 `;
 const GameTitle = styled.em`
```

Comment:
```suggestion
const GameTitle = styled.p or span`
```

em 태그 자체가 기본적으로 Italic 폰트로 바꿔줍니다.
그래서 태그를 바꾸면 돼요!